From b90c1918b9f4cb4ec9e111eed4195430a56f6e2e Mon Sep 17 00:00:00 2001
From: Lucian Zala <zala.lucian@gmail.com>
Date: Thu, 16 Feb 2017 20:42:47 +0200
Subject: [PATCH 086/107] staging: fsl-mc: fix coding style warning
Git-commit: e65a60e5cd14e2416bf5a1e1092ce1ae9defdef5
Patch-mainline: v4.12-rc1
References: bsc#1041022

Fix for "WARNING: Avoid multiple line dereference - prefer
'mc_msi_domain->host_data'" found by checkpatch.pl in
bus/irq-gic-v3-its-fsl-mc-msi.c.

Signed-off-by: Lucian Zala <zala.lucian@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Alexander Graf <agraf@suse.de>

---
 drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c b/drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c
index 87e4471..49127ac 100644
--- a/drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c
+++ b/drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c
@@ -95,8 +95,8 @@ int __init its_fsl_mc_msi_init(void)
 			continue;
 		}
 
-		WARN_ON(mc_msi_domain->
-				host_data != &its_fsl_mc_msi_domain_info);
+		WARN_ON(mc_msi_domain->host_data !=
+			&its_fsl_mc_msi_domain_info);
 
 		pr_info("fsl-mc MSI: %s domain created\n", np->full_name);
 	}
-- 
1.8.5.6

